recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -0,0 +1,8 @@
+[main]
+dns=systemd-resolved
+plugins=keyfile
+autoconnect-retries-default=0
```

Comment:
reading the documentation 0 seems unusual here. what does it mean? no retry?

Reply:
according to https://developer-old.gnome.org/NetworkManager/stable/settings-connection.html a value of 0 for autoconnect-retries means "retry forever".
interestingly this cannot be found in the documentation of NetworkManager.conf itself.